From c4e233a2821ed7af09c91e10edae80e32bc579cc Mon Sep 17 00:00:00 2001
From: Cristian Ciocaltea <cristian.ciocaltea@collabora.com>
Date: Wed, 4 Dec 2024 14:09:35 +0200
Subject: [PATCH 39/90] [WIP-YUV420] drm/rockchip: dw_hdmi_qp: Enable YUV420
 output format

Since both RK3576 and RK3588 SoCs are capable of handling YUV420 output
color format, add it to the platform supported formats.

Signed-off-by: Cristian Ciocaltea <cristian.ciocaltea@collabora.com>
---
 drivers/gpu/drm/rockchip/dw_hdmi_qp-rockchip.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/rockchip/dw_hdmi_qp-rockchip.c b/drivers/gpu/drm/rockchip/dw_hdmi_qp-rockchip.c
index 0e9719baa7..9994c5510c 100644
--- a/drivers/gpu/drm/rockchip/dw_hdmi_qp-rockchip.c
+++ b/drivers/gpu/drm/rockchip/dw_hdmi_qp-rockchip.c
@@ -599,7 +599,8 @@ static int dw_hdmi_qp_rockchip_bind(struct device *dev, struct device *master,
 
 	plat_data.supported_formats = BIT(HDMI_COLORSPACE_RGB) |
 				      BIT(HDMI_COLORSPACE_YUV444) |
-				      BIT(HDMI_COLORSPACE_YUV422);
+				      BIT(HDMI_COLORSPACE_YUV422) |
+				      BIT(HDMI_COLORSPACE_YUV420);
 
 	encoder = &hdmi->encoder.encoder;
 	encoder->possible_crtcs = drm_of_find_possible_crtcs(drm, dev->of_node);
-- 
2.46.0

